compiler/testSuite/Efloat05.bal

```diff
@@ -0,0 +1,8 @@
+import ballerina/io;
+
+public function main() {
+    if 0.0 / 0.0 === -0.0 / 0.0 { // @error
+        return;
+    }
+    io:println("impossible");
```

Comment:
@jclark
I think this should be an unreachable error but currently it's not because of `multiSemType`.
I am going to change this to not to produce a `multiSemType`, is that correct?

Reply:
No it's not an unreachable error, because type of === is not a singleton https://github.com/ballerina-platform/ballerina-spec/issues/902#issuecomment-887136996